include/libtorrent/alert_types.hpp

```diff
@@ -56,6 +56,9 @@ POSSIBILITY OF SUCH DAMAGE.
 #pragma GCC diagnostic ignored "-Wdeprecated-declarations"
 #endif
 
+// c++/cx keyword
+#undef interface
```

Comment:
because if I delete these two lines in the future it will still work (it will appear to work because I don't have any winRT CI environment). and the suspicious comment (which seems to be a contradiction) may make it more likely to be removed.

btw, how can you be so confident this works? I would be surprised if you can declare an interface after including this header. (but I don't know the details of c++/cx). If this is in fact correct, I think the comment need to explain why it doesn't break code that includes this header and then use the interface.. keyword.. or whatever it is.